internal/distributedtracing/distributedtracing.go

```diff
@@ -116,7 +116,8 @@ func Init(ctx context.Context, raw []byte, id string) (*otlptrace.Exporter, trac
 		trace.WithSampler(trace.ParentBased(trace.TraceIDRatioBased(float64(*distributedTracingConfig.SampleRatePercentage)/float64(100)))),
 		trace.WithSpanProcessor(trace.NewBatchSpanProcessor(traceExporter)),
 	)
-
+	otel.SetTracerProvider(traceProvider)
```

Comment:
This sets the global TracerProvider? I don't think we should do that. Maybe we can export it instead, and have it be set in the opa-envoy-plugin's initialization routines?

I'm wondering because we don't want to mess with the globals in case someone is embedding OPA, and using OTel, too.

Reply:
Sorry for the quiet weekend. I have made some changes to export only `TracerProvider` through `plugin.Manager`. And `DistributedTracingOpts` can create from `TracerProvider`.